{
  "ok": true,
  "issues": [],
  "zip": "NRW_SYNC_20250825-182219.zip",
  "sha256_actual": "ccca1cf9c5f4da9f5506dc1d2bd278579a5b900490e5fd4eb09423746a7f38c1",
  "sha256_expected": "ccca1cf9c5f4da9f5506dc1d2bd278579a5b900490e5fd4eb09423746a7f38c1",
  "preflight_stdout": "",
  "preflight_stderr": "/Users/hadrianbelove/Downloads/new-release-wall/.venv/bin/python: can't open file '/Users/hadrianbelove/Downloads/new-release-wall/restore/NRW_SYNC_20250825-182219/restore/NRW_SYNC_20250825-182219/tools/preflight.py': [Errno 2] No such file or directory",
  "restored_preflight_ok": true
}